src/main/java/com/epam/ta/reportportal/ws/model/StartTestItemRQ.java

```diff
@@ -47,7 +44,7 @@
 	@JsonProperty(value = "uniqueId")
 	private String uniqueId;
 
-	@NotNull
+	@NotBlank
```

Comment:
https://github.com/hibernate/hibernate-validator/blob/b173247ab36545b90f2bf64a3838f39f4aab2eb1/engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/hv/NotBlankValidator.java#L33

Do we want to allow null values everywhere where NotEmpty and NotNull were?

Reply:
I checked with null and values. It works properly